package.json

```diff
@@ -9,7 +9,6 @@
     "bundlemon": "^1.4.0",
     "eslint": "^8.23.1",
     "eslint-config-mourner": "^2.0.3",
-    "git-rev-sync": "^3.0.2",
```

Comment:
The `git-rev-sync` package relies on some CommonJS features that are not available in ESM mode, so I've replaced it with a more modern package.